src/main/scala/com/danieljharvey/eggserver/LevelsLoader.scala

```diff
@@ -1,16 +1,60 @@
 package com.danieljharvey.eggserver
 
 import scala.util.Try
+import net.liftweb.json._
+import net.liftweb.json.Serialization.write
+
+case class BoardSize (width: Int, height: Int)
+
+// type that json will turn into
+case class JSONTile (
+    action: Option[String],
+    background: Option[Boolean],
+    breakable: Option[Boolean],
+    collectable: Option[Int],
+    createPlayer: Option[String],
+    dontAdd: Option[Boolean],
+    frontLayer: Option[Boolean],
+    id: Option[Int],
+    title: Option[String],
+    x: Option[String],
+    y: Option[String]
+)
+
+// actual nice type with no options and defaults instead
```

Comment:
Typically we don't need two types to represent the same thing. Let's chat about it sometime why you ended up doing it like this.